File: testnesne/src/Watch.java
public class Watch {

    public String Brand;
    public String Type;
    public int Pressure;
    private int price;

    public void setBrand(String Brand) {
        this.Brand = Brand;
    }

    public String getBrand() {

        return Brand;
    }

    public String getType() {
        return Type;
    }

    public void setType(String Type) {
        this.Type = Type;
    }
public void setPressure(int Pressure){
        this.Pressure=Pressure;
}
    public int getPressure(){
        return Pressure;
    }

    public void setPrice(int price){
        this.price=price;
    }

    public int getPrice(){
        return price;
    }
    public Watch(){

    }


    //public Watch (String Type,int Pressure,String Brand,int price){

        //this.price=price;
       // this=price
    //}
}

File: NesneTabanli/Watch.cs
using System;
namespace NesneTabanli
{
	public class Watch
	{
        public string Brand;
        public string Type;
        public int Pressure { get; set; }
        private int price;



        public void setBrand(string Brand)
        {
            this.Brand = Brand;

            Watch.Watchs saats = new Watch.Watchs();

           

            switch (this.Brand)

            {
                case "Seiko":

                    this.price = 12800;
                    Console.WriteLine(this.price);

                    break;

            }

        }
        public void setType(string Type)
        {
            this.Type = Type;

        }
        public void setprice(int price)
        {
            this.price =price;

        }


        public string getBrand()
        {
            return Brand;
        }
        public string getType()
        {
            return Type;
        }
        public int getprice()
        {
            return price;
        }

        public enum Watchs
        {
            seiko = 1,
            tissot =2,
        

        }

           



    }
}

